src/equations/inviscid_burgers_1d.jl

```diff
@@ -100,6 +136,26 @@ function flux_ec(u_ll, u_rr, orientation, equation::InviscidBurgersEquation1D)
 end
 
 
+# See https://metaphor.ethz.ch/x/2019/hs/401-4671-00L/literature/mishra_hyperbolic_pdes.pdf ,
```

Comment:
If we add code, we should also have tests for it

Reply:
I added some consistency tests for both Godunov flux & Engquist-Osher.